saleor/graphql/api.py

```diff
@@ -173,6 +175,36 @@ def serialize_webhook_event(value):
         graphql.DirectiveLocation.OBJECT,
     ],
 )
+
+
+def load_webhook_types():
+    """Load webhooks types so they can be used in the schema.
+
+    This function inspects webhooks by iterating over subclasses of `AsyncWebhookBase`
+    and `SyncWebhookBase` classes, and then imports the subscription type for each
+    webhook.
+    """
+    from ..webhook.base import AsyncWebhookBase, SyncWebhookBase
+
+    def load_webhooks_for_base_cls(base_cls):
+        for cls in base_cls.__subclasses__():
```

Comment:
How will it work in case of having more advanced relations like:
```
AsyncWebhookBase -> AsyncOrderWebhookBase -> OrderCreated
```
?
Wouldn't be simpler to have something like `@register` that would be used to decorate the webhooks that should be included? 
```python
@register
class OrderCreated(AsyncOrderWebhookBase):
...
```

Reply:
I tried the solution with `register` but it didn't work well with loading modules into the GraphQL schema - some "register" calls were running after the schema was created, so the subscription was not added to the schema. I think it's related to order of imports but didn't debug this deeper. For now I thought this solution should be enough, and when we need to add such case as in your example, then this function can be easily updated.